dts/riscv32/riscv32-fe310.dtsi

```diff
@@ -0,0 +1,178 @@
+/ {
```

Comment:
Need status properties on the peripheral nodes.  The typical pattern is that peripherals are status = "disabled" by default, and the board.dts explicitly enables. them with a status = "ok".

Reply:
No interrupt controllers are typically always enabled, so we don't bother with status on them.  Its more things that have external pin/signals that would be board/application specific.  While interrupts can obviously fall in that group, its almost always the case that at least one internal interrupt ends being needed.